community/kernel/src/main/java/org/neo4j/storageengine/api/StorageStatement.java

```diff
@@ -112,9 +113,10 @@
      -     */
     Cursor<RelationshipItem> relationshipsGetAllCursor();
 
-    Cursor<PropertyItem> acquirePropertyCursor( long propertyId, Lock shortLivedReadLock );
+    Cursor<PropertyItem> acquirePropertyCursor( long propertyId, Lock shortLivedReadLock, AssertOpen assertOpen );
```

Comment:
Original PR introduced `assertOpen` checks for `acquireSingleNodeCursor `, `acquireSingleRelationshipCursor`, etc why we do not need those anymore here? (and related stack)